gems/actionpack/CVE-2021-22885.yml

```diff
@@ -64,4 +64,5 @@ patched_versions:
   - "~> 5.2.4.6"
   - "~> 5.2.6"
   - "~> 6.0.3.7"
```

Comment:
Instead, could we do:
`- "~> 6.0.3, >= 6.0.3.7"`

Reply:
👍 Updated in [`3f198f2`](https://github.com/rubysec/ruby-advisory-db/compare/36785ac88f0c8ef4ee9ba0c6527d65694bb67a25..3f198f2b13dce1f72e03393d4d935a9cbce121fd).